pkg/byteorder/byteorder_bigendian.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build arm arm64 mips mips64 ppc64
```

Comment:
Can't arm be programmed to work in BE mode?

Reply:
AFAIK, gccgo already supports `armbe`and `arm64be`. Not sure whether Cilium builds with gccgo, though :smile: 

But as @twpayne already stated, the big endian architectures will use distinct `GOARCH` values. `arm` and `arm64` always specify the little endian ARM architectures. Also see:

https://github.com/golang/sys/blob/e8d321eab015fdac193488ca80676aafc248f81d/unix/endian_big.go#L5-L6

and

https://github.com/golang/sys/blob/e8d321eab015fdac193488ca80676aafc248f81d/unix/endian_little.go#L5-L6